pages/shoppingcart.js

```diff
@@ -0,0 +1,9 @@
+function ShoppingCart() {
+  return (
+    <div>
```

Comment:
There is only one element nested inside of your `div` element. Therefore, the `div` is not needed and you can remove it.